tests/unit/test_okta_plugin.py

```diff
@@ -169,6 +169,27 @@ def test_no_cached_token(mocker, mock_plugin_service, mock_session, mock_func, m
     assert WrapperProperties.USER.get(test_props) == _DB_USER
     assert WrapperProperties.PASSWORD.get(test_props) == _TEST_TOKEN
 
+@patch("aws_advanced_python_wrapper.okta_plugin.OktaAuthPlugin._token_cache", _token_cache)
+def test_no_cached_token_raises_exception(mocker, mock_plugin_service, mock_session, mock_func, mock_client, mock_dialect, mock_credentials_provider_factory):
+    test_props: Properties = Properties({"plugins": "okta", "user": "postgresqlUser", "idp_username": "user", "idp_password": "password"})
+
```

Comment:
```suggestion
    test_props: Properties = Properties(
        {"plugins": "federated_auth", "user": "postgresqlUser", "idp_username": "user", "idp_password": "password"})
    WrapperProperties.DB_USER.set(test_props, _DB_USER)

```

Reply:
Is there a reason why we are using 'federated_auth' instead of 'okta' for the plugin in a okta unit test?

Reply:
I left the comments on the wrong file, meant to leave it on the fed auth test. Comments applicable to both